projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.ts

```diff
@@ -121,4 +129,10 @@ export class CaseFileViewFieldComponent implements OnInit, AfterViewInit, OnDest
       this.categoriesAndDocumentsSubscription.unsubscribe();
     }
   }
+
+  public isIcpEnabled(icpEnabled:boolean, icp_jurisdictions:string[]): boolean {
+    return icpEnabled && icp_jurisdictions.includes(
```

Comment:
It would be a good idea to check if icp_jurisdictions is empty - if it is that means that all jurisdictions are allowed. Something like:
isIcpEnabled && ((!icp_jurisdictions?.length < 1) || icp_jurisdictions.includes( ...